dev/ci/ci-common.sh

```diff
@@ -25,12 +25,16 @@ git_checkout()
   local _URL=${2}
   local _DEST=${3}
 
+  # Allow an optional 4th argument for the commit
+  local _COMMIT=${4:-FETCH_HEAD}
+  local _DEPTH=${5:-1}
```

Comment:
Indeed, I am open to see how we can improve this, I couldn't get a good bash code for your first suggestion.